release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -408,6 +408,22 @@ Kubernetes 1.22 introduces link:https://kubernetes.io/docs/reference/using-api/d
 When you uninstall an Operator by using the web console, you can now remove Operands managed by the Operator automatically, including custom resources (CRs). For more information, see
 xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-web-console_olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster using the web console].
 
+[id="ocp-4-9-olm-admin-error-reporting"]
+==== Enhanced error reporting for cluster administrators
+Because administrators should not require understanding of the interaction process between the various low-level APIs or access to the OLM pod logs to successfully debug such issues, {product-title} {product-version} introduces the following enhancements in Operator Lifecycle Manager (OLM) to provide administrators with more comprehensible error reporting and messages:
```

Comment:
A few comments:

s/ understanding / an understanding? Probably not necessary. 

Your first instance of OLM is abbreviated, and the second instance is spelled out. This should probably be switched so that it's spelled out first, then abbreviated second.